synapse/lib/storm.py

```diff
@@ -4592,6 +4595,17 @@ async def execStormCmd(self, runt, genr):
 
             runt.layerConfirm(('node', 'del', node.form.name))
 
+            if deledges:
+                edges = []
+
+                async for (verb, n2iden) in node.iterEdgesN2():
+                    runt.layerConfirm(('node', 'edge', 'del', verb))
+                    edges.append((verb, n2iden))
+
+                for (verb, n2iden) in edges:
+                    n2 = await self.runt.snap.getNodeByBuid(s_common.uhex(n2iden))
+                    await n2.delEdge(verb, node.iden())
```

Comment:
Grabbing a SnapEditor and saving these in chunks might be good (there's probably other places we could do that too)